packages/webapp/src/shared/utils/storybook.tsx

```diff
@@ -52,8 +52,8 @@ import { WrapperProps, getWrapper } from '../../tests/utils/rendering';
  *
  */
 export function withProviders(wrapperProps: WrapperProps = {}) {
-  return (StoryComponent: StoryFn) => {
-    const { wrapper: WrapperComponent } = getWrapper(ApiTestProviders, wrapperProps) as any;
+  return (StoryComponent: StoryFn, storyContext: any) => {
```

Comment:
If the type form `import { StoryContext } from '@storybook/react';` does not match?

`storyContext: any` -> `storyContext: StoryContext`